components/ParallaxScroll.js

```diff
@@ -0,0 +1,25 @@
+export default class Parallax extends React.Component {
+  componentDidMount () {
+    window.addEventListener('scroll', this.updateParallax, false)
```

Comment:
The parallax effect will not trigger when the page is first load until the user scrolls. We need to also add this:

```js
    this.updateParallax()
```